src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -32,6 +32,11 @@ const resolveSHMDLink = environment.isProduction()
   ? 'https://veteran.apps.va.gov/smhdWeb'
   : 'https://veteran.apps-staging.va.gov/smhdWeb';
 
+// Oracle Health 'My VA Health' Portal
+const myVAHealthPortalLink = environment.isProduction()
+  ? 'https://patientportal.myhealth.va.gov/pages/home'
```

Comment:
Another way to do this: 
`${cernerEnvPrefixes[environment.BUILDTYPE]}patientportal.myhealth.va.gov`
or using `EXTERNAL_REDIRECTS`. There seems to be already defined constants with this that you could reuse.